node/node.go

```diff
@@ -1431,6 +1436,7 @@ func (app *App) grpcService(svc grpcserver.Service, lg log.Log) (grpcserver.Serv
 		return service, nil
 	case grpcserver.Post:
 		service := grpcserver.NewPostService(app.addLogger(PostServiceLogger, lg).Zap())
+		service.AllowConnections(app.Config.SMESHING.CoinbaseAccount != "")
```

Comment:
Perhaps it would be nice for the users to log a warning that a coinbase is not set and the node will not smesh. It might save somebody trouble. :) 

Alternatively,  would it be fine to shut down the node with a proper message? Enabling the PostService endpoint suggests that the user wants to smesh (create ATXs). It requires a coinbase, so very well we could simply abort here. This way the user won't miss important information that this setup is technically broken.

Reply:
> Perhaps it would be nice for the users to log a warning that a coinbase is not set

I can add a log. Not having a coinbase is a valid configuration. In a supervised setup the coinbase can be set via grpc on the `StartSmeshing` call, in a multi-smeshing setup the node is then just a non-smeshing node.